packages/components/src/RichTextEditor/utils/plugins/LinkManager/components/LinkModal/LinkModal.tsx

```diff
@@ -45,14 +46,22 @@ export const LinkModal = ({
       onDismiss={onDismiss}
       onAfterLeave={onAfterLeave}
       onAfterEnter={() => inputRef.current?.focus()}
+      data-testid="modal-test"
```

Comment:
Could this be added in the test instead of the component?

Reply:
Oh I should actually remove this 😆 its a left over when I was trying to find a good way to test this